config/environments/production.rb

```diff
@@ -83,8 +85,12 @@
   config.log_formatter = ::Logger::Formatter.new
 
   # Specify that we're not at the root
-  config.action_controller.relative_url_root = '/app/ppd'
-  config.relative_url_root = '/app/ppd'
+  config.action_controller.relative_url_root = ENV['RELATIVE_URL_ROOT'] || '/app/ppd'
```

Comment:
Looking at this, I'm wondering now if it would make more sense to either make the default `/`, or to assume that the `RELATIVE_URL_ROOT` will always be passed in as a param when running in production mode? Having a baked-in default of `/app/ppd` looks out of place here now